code/game/objects/items/mantis.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/mantis/blade
+	name = "mantis blade"
+	desc = "Powerful inbuilt blade, hidden just beneath the skin. Singular brain signals directly link to this bad boy, allowing it to spring into action in just seconds."
```

Comment:
"A blade designed to be hidden just beneath the skin. The brain is directly link to this bad boy, allowing it to spring into action."

You don't  have to use it, but I needed to reread that description.

Reply:
It seems that this is the limit of my English skills? I don't really get what you mean...?

Reply:
I am giving you a new description to use. You do not have to use it, but read both comments I have made and see if you like them. I hope this clears up any miss understandings.